scripts/build_binary.sh

```diff
@@ -35,6 +35,10 @@ if [[ -n "${3}" ]]; then
 fi
 
 GOOS=${TARGET_GOOS:-} GOARCH=${TARGET_GOARCH:-} CGO_ENABLED=0 \
-       	go build -installsuffix cgo -a -ldflags "-s ${version_ldflags}" \
-       	-o ../$1/docker-credential-ecr-login \
-	./cli/docker-credential-ecr-login
+        go build \
+        -installsuffix cgo \
+        -a \
+        -ldflags "-buildid= -s ${version_ldflags}" \
```

Comment:
Explicitly adding `-buildid=` removes any buildid from being added to the binary, which fixes the issue of go build-id style mismatch. I don't think we have any use for build-id, so think this will work. If my understanding is wrong, we can also explicitly pass a GNU style buildid using `-B` option.